examples/storage/couchdb/images/init_couch/configure_couch.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+fqdn=$(hostname --fqdn)
+
+cp /core-config/default.ini $INSTALL_LOCATION/etc/default.ini
+cp /core-config/local.ini $INSTALL_LOCATION/etc/local.ini
+cp /core-config/vm.args $INSTALL_LOCATION/etc/vm.args
+mkdir $INSTALL_LOCATION/etc/local.d
+mkdir $INSTALL_LOCATION/etc/default.d
+sed -i $INSTALL_LOCATION/etc/vm.args -e "s/^\-name couchdb/\-name couchdb@$fqdn/"
```

Comment:
Is this line required? Typically Erlang will assume you want the FQDN if you use `-name` and leave the FQDN blank.